packages/cryptoplease_link/lib/src/payments/escrow_account.dart

```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: deprecated_member_use
```

Comment:
What is deprecated?

Reply:
We need it since `CompiledMessage.fromSignedTransaction` is marked as deprecated in `solana: ^0.26.1`. Had to update it to use `SignedTx.decode` [here](https://github.com/cryptoplease/cryptoplease-dart/pull/632/files#diff-42c44042a8e0fb7a000a7bac97bf5247bdb974e99f61279d04efb00c83a316ea)

Reply:
You can use `SignedTx.fromBytes`. It doesn't make sense to deprecate this error since it's a package controlled by us.